queries.sql

```diff
@@ -0,0 +1,24 @@
+
+-- Find all animals whose name ends in "mon".
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+-- List the name of all animals born between 2016 and 2019.
+SELECT * FROM animals WHERE date_of_birth BETWEEN DATE '2016-01-01' AND '2019-12-31';
+
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you update this query to exactly answer the asked question as the current one exclude animals born on  **'2016-01-01'** AND **'2019-12-31'**.
You can change it to something like this:
```PostgreSQL
... WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';
```

Reply:
Noted with thanks